src/WebJobs.Script.Abstractions/WebJobs.Script.Abstractions.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\..\build\common.props" />
     <PropertyGroup>
-    <Version>1.0.3-preview</Version>
```

Comment:
We've never even released v1 and it's still in preview. Seems strange to be bumping the major version of it if it's never been released. @fabiocav for visibility

Reply:
I recommended bumping major version as  Logic Apps team has dependency on this package.

Reply:
I think we should just increment this normally to 1.0.4-preview. Bumping the major version wouldn't prevent issues if extensions are actually using this I think. E.g. say an extension is using the existing 1.0.3 version and setting a retry. An updated host using version 2.0.0 wouldn't see/apply that retry anymore (breaking). Depending on how assembly unification works in this case, the issue might even be worse - if the extension unifies to the host version, the extension would see missing method exceptions?